chapters/hpmor-chapter-064.tex

```diff
@@ -10,13 +10,14 @@
 \makeatother
 
 \newcommand{\OmakeIVsection}[2][1.5]{%
-\OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
+  \OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
```

Comment:
For PR #66?

Reply:
good point, looks I modified it in both, now copied there too